app/controllers/Model.java

```diff
@@ -0,0 +1,265 @@
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            ResultSet result = statement.executeQuery(sql);
+            if(result.next()) {
+                New = false;
+            }
+
+            result.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            statement.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            connection.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        if(New) {
+            insert(true);
+        } else {
+            update();
+        }
+    }
+
+    final void get(int id) throws NoSuchFieldException, IllegalAccessException {
+        if(db == null) {
+            throw new NullDataBaseExcepion();
+        }
+
+        Connection connection = db.getConnection();
+        String sql;
+        sql = "SELECT * FROM " + this.getClass().getSimpleName().toLowerCase()
+                + " where id = " + id;
+        Statement statement = null;
+
+        try {
+            statement = connection.createStatement();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            ResultSet result = statement.executeQuery(sql);
+
+            if(!result.next()) {
+                throw new NoSuchElementException();
+            } else {
+                Field[] fields = this.getClass().getDeclaredFields();
+
+                for(Field field : fields) {
+                    field.set(this, result.getObject(makeSql(field.getName())));
+                }
+
+            }
+
+            result.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            statement.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            connection.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    final void insert(boolean New) throws IllegalAccessException, NoSuchFieldException {
+        Connection connection = db.getConnection();
+        String sql;
+        sql = "INSERT INTO " + this.getClass().getSimpleName().toLowerCase() + " VALUES ";
+
+        sql = prepareBrackets(sql);
+
+        PreparedStatement statement = null;
+
+        try {
+            statement = connection.prepareStatement(sql);
+
+            Field[] fields = this.getClass().getDeclaredFields();
+
+            int index = 0;
+            for(Field field : fields) {
+                if(New && index == 0) {
+                    statement.setObject(1, findMaxId()+1);
+                    index++;
+                    continue;
+                }
+                statement.setObject(++index, field.get(this));
+            }
+
+            System.out.println(statement.toString());
```

Comment:
I don't think we need this line, you can use some kind of logger though if you wish